File: daily/Nov30.cc
#include <iostream>
using namespace std;

class Solution {
 public:
  bool canAlignWin(vector<int>& nums) {
    int oneSum = 0, twoSum = 0;
    for (int num : nums) {
      if (num > 0 && num < 10) {
        oneSum += num;
      } else {
        twoSum += num;
      }
    }
    return (oneSum == twoSum) ? false : true;
  }
};

File: daily/Nov30.rs
impl Solution {
    pub fn can_alice_win(nums: Vec<i32>) -> bool {
        let (mut oneSum, mut twoSum) = (0, 0);
        for num in &nums {
            if *num > 0 && *num < 10 {
                oneSum += *num;
            } else {
                twoSum += *num;
            }
        }
        if oneSum != twoSum {
            return true;
        }
        false
    }
}
